src/config.ts

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import path from 'path'
+import url from 'url'
+import { inspect } from 'util'
+// Load all the ENV variables from `.env`, then `.env.local`, into process.env
+import dotenv from 'dotenv-flow'
+dotenv.config()
+
+/**
+ * Database
+ */
+const db: any = {
+  username: process.env.MYSQL_USER ?? 'server',
+  password: process.env.MYSQL_PASSWORD ?? 'password',
+  database: process.env.MYSQL_DATABASE ?? 'xrchat',
+  host: process.env.MYSQL_HOST ?? 'localhost',
+  port: process.env.MYSQL_PORT ?? 3306,
+  dialect: 'mysql',
+  forceRefresh: process.env.FORCE_DB_REFRESH === 'true'
+}
+db.url = process.env.MYSQL_URL ??
+  `mysql://${db.username}:${db.password}@${db.host}:${db.port}/${db.database}`
+
+/**
+ * Server / backend
+ */
+const server: any = {
+  mode: process.env.SERVER_MODE ?? 'media',
+  hostname: process.env.SERVER_HOSTNAME ?? 'localhost',
+  port: process.env.SERVER_PORT ?? 3030,
+  // Public directory (used for favicon.ico, logo, etc)
+  publicDir: process.env.SERVER_PUBLIC_DIR ?? path.resolve(__dirname, '..', 'public'),
+  // Used for CI/tests to force Sequelize init an empty database
+  performDryRun: process.env.PERFORM_DRY_RUN ?? false,
```

Comment:
a major pitfall in env vars is that they are always strings, 
so if 
`export PERFORM_DRY_RUN=false`
it is loaded as a string "false"
`if ("false") true; else false; // returns true`

so if we define this default value in .env and we want it to be false
it muse be like this:
`export PERFORM_DRY_RUN=`
empty string is false in JS

another option is to rewrite this code to be 
`performDryRun: process.env.PERFORM_DRY_RUN=="true" ?? false,`

Note: same issue will happen with SERVER_PORT but JS automatic coersion will fix that automatically.

Reply:
Ahh right thanks! Will update.

Reply:
Ok, check, will fix.

Reply:
Fixed.